localstack/services/sqs/constants.py

```diff
@@ -30,3 +30,11 @@
     QueueAttributeName.LastModifiedTimestamp,
     QueueAttributeName.QueueArn,
 ]
+
+# URL regexes for various endpoint strategies
+STANDARD_STRATEGY_URL_REGEX = r"sqs.(?P<region_name>[a-z0-9-]{1,}).localhost.localstack.cloud:\d{1,}\/(?P<account_id>\d{12})\/(?P<queue_name>[a-zA-Z0-9_-]+(.fifo)?)$"
+DOMAIN_STRATEGY_URL_REGEX = r"(?P<region_name>[a-z0-9-]{1,}).queue.localhost.localstack.cloud:\d{1,}\/(?P<account_id>\d{12})\/(?P<queue_name>[a-zA-Z0-9_-]+(.fifo)?)$"
```

Comment:
It seems that the Domain strategy regex does not allow the special case of the omitted us-east-1 region. This small correction should do the trick.
`((?P<region_name>[a-z0-9-]{1,}).)?queue.localhost.localstack.cloud:\d{1,}\/(?P<account_id>\d{12})\/(?P<queue_name>[a-zA-Z0-9_-]+(.fifo)?)$`
Also it might probably be good to add a test case in `test_is_sqs_queue_url` for this.

Reply:
Good catch, fixed :+1: